packages/x-components/tests/e2e/cucumber/recommendations/recommendations.feature

```diff
@@ -2,10 +2,12 @@ Feature: Recommendations component
 
   Scenario Outline:  1. Recommendations are displayed
     Given following config: max items to store is <maxItemsToRequest>
+    And   suggestions response being mock "<mock>"
+    Then  number of rows requested is <maxItemsToRequest>
     Then  number of displayed recommendations are equal or less than <maxItemsToRequest>
     And   each recommendation has an associated hyperlink containing image and text
 
     Examples:
-      | maxItemsToRequest |
-      | 7                 |
-      | 160               |
+      | maxItemsToRequest | mock    |
+      | 3                 | default |
```

Comment:
If you end up refactoring the tests using a `named export` instead of the `default` one, remember to update these values here.